bitnami/wordpress/values.yaml

```diff
@@ -446,6 +446,13 @@ ingress:
   ##
   path: /
 
+  ## Alternative host names. This will add SANs (Subject Alternative Names) to the cert so
+  ## that one SSL certificate covers multiple names. The hosts here will map to ingress.path.
+  ##
+  ## If you want a separate certificate, use ingress.extraTls.hosts, ingress.extraTls.secretName
+  ## for the cert, and ingress.extraHosts.name and ingress.extraHosts.path for the ingress mapping
+  alternativeHostNames: []
```

Comment:
Please, remove this

Reply:
Ah yes, I missed that sorry. Will remove

Reply:
This is now removed from latest commit